package.json

```diff
@@ -33,6 +33,7 @@
   },
   "devDependencies": {
     "@carbon/bundler": "^10.9.0",
+    "@carbon/grid": "^10.17.0",
```

Comment:
@vladbalanescu Should this be a production dependency in `cdai/package.json`?

Reply:
@SimonFinney Done, mind re-reviewing?